packages/blue-sdk/src/addresses.ts

```diff
@@ -106,6 +106,21 @@ export const addresses = {
     verUsdc: "0x59aaF835D34b1E3dF2170e4872B785f11E2a964b",
     testUsdc: "0xBC77067f829979812d795d516E523C4033b66409",
   },
+  [ChainId.PolygonMainnet]: {
+    morpho: "0x1bF0c2541F820E775182832f06c0B7Fc27A25f67",
+    bundler3: {
+      bundler3: "0x2d9C3A9E67c966C711208cc78b34fB9E9f8db589",
+      generalAdapter1: "0xB261B51938A9767406ef83bbFbaAFE16691b7047",
+      paraswapAdapter: "0x5F2617F12D1fDd1e43e72Cb80C92dFcE8124Db8d",
+    },
+    permit2: "0x000000000022D473030F116dDEE9F6B43aC78BA3",
+    adaptiveCurveIrm: "0xe675A2161D4a6E2de2eeD70ac98EEBf257FBF0B0",
+    publicAllocator: "0xfac15aff53ADd2ff80C2962127C434E8615Df0d3",
+    metaMorphoFactory: "0xa9c87daB340631C34BB738625C70499e29ddDC98",
+
+    wNative: "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
+    usdc: "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
```

Comment:
This is the bridged USDC version. The native USDC is https://polygonscan.com/token/0x3c499c542cef5e3811e1192ce70d8cc03d5c3359. Although the bridged has higher total supply.

Reply:
I will simply remove it then, to make sure we don't use it in a bad way